client/state/test/index.js

```diff
@@ -12,7 +12,39 @@ import { useSandbox } from 'test/helpers/use-sinon';
 import { createReduxStore } from '../';
 import currentUser from 'state/current-user/reducer';
 
+const should = chai.should();
+
 describe( 'index', () => {
+	describe( 'addReducer', () => {
+		const name = 'myReducer';
+		const func = ( state ) => state || {};
+
+		it( 'should add and remove a reducer by name', () => {
+			const store = createReduxStore();
+
+			should.not.exist( store.getReducers()[ name ] );
```

Comment:
Because I started out using `should`, and then used `expect` by your recommendation. ;) I'll consolidate.